### ISSUE TITLE
Translator transchoice not working v6

### ISSUE THREAD

**Issue by Siqu** [pimcore/pimcore#4603]
# Bug Report

### Expected behavior

Pimcore Translator follow the new pattern for the Translator. It now uses the trans function with a additional parameter to handle translation choices. Internally the Translator now uses a IntlFormatterInterface service which follows the ICU MessageFormat to handle the translation via choices. See [Message Format Symfony documentation](https://symfony.com/doc/current/translation/message_format.html).

The transChoice function is now deprecated since Symfony 4.2.

### Actual behavior

Pimcore Translator follows the new pattern.

**Comment by brusch** [comment]
You're writing that the expected behavior (new pattern) is equal to the actual behavior (new pattern), so what's your point? 
Thanks for clarifying.

**Comment by brusch** [comment]
Please also provide meaningful examples. Thanks

**Comment by brusch** [comment]
```php
$trans->transChoice(
            '{0} There are no apples|{1} There is one apple|]1,Inf[ There are %count% apples',
            10,
            ['%count%' => 10]
        );
```

Working as expected ...

**Comment by Siqu** [comment]
That is strange i retrieve the error that IdentityTranslator->selector has no method choose. 

I check the exact error once i am at the office again.

**Comment by Siqu** [comment]
Yes the problem is following:

Pimcore\Translation\Translator calls on line 154
```
$term = $this->selector->choose($term, (int) $number, $locale);
```

Symfony\Component\Translation\IdentityTranslator has no method choose.

**Comment by dpfaffenbauer** [comment]
already fixed in 6.0.1
